cpu-bind=MASK - bp1-compute019, task  0  0 [3374448]: mask 0xfffc000fffc00 set
Traceback (most recent call last):
  File "/user/home/zj22662/photon-net/calc_test2.py", line 94, in <module>
    run_evaluation(hard_predict=False)
  File "/user/home/zj22662/photon-net/calc_test2.py", line 89, in run_evaluation
    iterate_models(output_folder, results_subfolder, model_number, input_config, hard_predict)
  File "/user/home/zj22662/photon-net/calc_test2.py", line 64, in iterate_models
    probs = tuple(probs)
            ^^^^^^^^^^^^
TypeError: 'int' object is not iterable
